Description: Upstream changes introduced in version 0.8.0~rc1-4~bpo60+1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 lxc (0.8.0~rc1-4~bpo60+1) squeeze-backports; urgency=low
 .
   * Backport for Debian/Squeeze.
   * Remove usage of DEB_HOST_MULTIARCH.
 .
 The person named in the Author field signed this changelog entry.
Author: Sven Hoexter <hoexter@debian.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- /dev/null
+++ lxc-0.8.0~rc1/config/Makefile
@@ -0,0 +1,343 @@
+# Makefile.in generated by automake 1.11.1 from Makefile.am.
+# config/Makefile.  Generated from Makefile.in by configure.
+
+# Copyright (C) 1994, 1995, 1996, 1997, 1998, 1999, 2000, 2001, 2002,
+# 2003, 2004, 2005, 2006, 2007, 2008, 2009  Free Software Foundation,
+# Inc.
+# This Makefile.in is free software; the Free Software Foundation
+# gives unlimited permission to copy and/or distribute it,
+# with or without modifications, as long as this notice is preserved.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY, to the extent permitted by law; without
+# even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.
+
+
+
+pkgdatadir = $(datadir)/lxc
+pkgincludedir = $(includedir)/lxc
+pkglibdir = $(libdir)/lxc
+pkglibexecdir = $(libexecdir)/lxc
+am__cd = CDPATH="$${ZSH_VERSION+.}$(PATH_SEPARATOR)" && cd
+install_sh_DATA = $(install_sh) -c -m 644
+install_sh_PROGRAM = $(install_sh) -c
+install_sh_SCRIPT = $(install_sh) -c
+INSTALL_HEADER = $(INSTALL_DATA)
+transform = $(program_transform_name)
+NORMAL_INSTALL = :
+PRE_INSTALL = :
+POST_INSTALL = :
+NORMAL_UNINSTALL = :
+PRE_UNINSTALL = :
+POST_UNINSTALL = :
+build_triplet = x86_64-pc-linux-gnu
+host_triplet = x86_64-pc-linux-gnu
+subdir = config
+DIST_COMMON = $(srcdir)/Makefile.am $(srcdir)/Makefile.in compile \
+	config.guess config.sub depcomp install-sh missing
+ACLOCAL_M4 = $(top_srcdir)/aclocal.m4
+am__aclocal_m4_deps = $(top_srcdir)/config/acinclude.m4 \
+	$(top_srcdir)/config/linux.m4 $(top_srcdir)/configure.ac
+am__configure_deps = $(am__aclocal_m4_deps) $(CONFIGURE_DEPENDENCIES) \
+	$(ACLOCAL_M4)
+mkinstalldirs = $(install_sh) -d
+CONFIG_HEADER = $(top_builddir)/src/config.h
+CONFIG_CLEAN_FILES =
+CONFIG_CLEAN_VPATH_FILES =
+SOURCES =
+DIST_SOURCES =
+DISTFILES = $(DIST_COMMON) $(DIST_SOURCES) $(TEXINFOS) $(EXTRA_DIST)
+ACLOCAL = ${SHELL} /home/sven/lxc-0.8.0~rc1/config/missing --run aclocal-1.11
+AMTAR = ${SHELL} /home/sven/lxc-0.8.0~rc1/config/missing --run tar
+AUTOCONF = ${SHELL} /home/sven/lxc-0.8.0~rc1/config/missing --run autoconf
+AUTOHEADER = ${SHELL} /home/sven/lxc-0.8.0~rc1/config/missing --run autoheader
+AUTOMAKE = ${SHELL} /home/sven/lxc-0.8.0~rc1/config/missing --run automake-1.11
+AWK = mawk
+BINDIR = /usr/bin
+CAP_LIBS = -lcap
+CC = gcc
+CCDEPMODE = depmode=none
+CFLAGS = -g -O2 -Wall
+CPP = gcc -E
+CPPFLAGS = 
+CYGPATH_W = echo
+DATADIR = /usr/share
+DEFS = -DHAVE_CONFIG_H
+DEPDIR = .deps
+DOCDIR = /usr/share/doc/lxc
+ECHO_C = 
+ECHO_N = -n
+ECHO_T = 
+EGREP = /bin/grep -E
+EXEEXT = 
+GREP = /bin/grep
+INCLUDEDIR = /usr/include
+INSTALL = /usr/bin/install -c
+INSTALL_DATA = ${INSTALL} -m 644
+INSTALL_PROGRAM = ${INSTALL}
+INSTALL_SCRIPT = ${INSTALL}
+INSTALL_STRIP_PROGRAM = $(install_sh) -c -s
+LDFLAGS = 
+LIBDIR = /usr/lib/lxc
+LIBOBJS = 
+LIBS = 
+LINUX_DIR = default
+LINUX_SRCARCH = x86
+LOCALSTATEDIR = /var
+LTLIBOBJS = 
+LXCINITDIR = ${libexecdir}/lxc
+LXCPATH = ${localstatedir}/lib/lxc
+LXCROOTFSMOUNT = ${prefix}/lib/lxc
+LXCTEMPLATEDIR = /usr/share/lxc/templates
+LXC_GENERATE_DATE = Fri Jul 6 13:43:02 UTC 2012
+MAKEINFO = ${SHELL} /home/sven/lxc-0.8.0~rc1/config/missing --run makeinfo
+MKDIR_P = /bin/mkdir -p
+OBJEXT = o
+PACKAGE = lxc
+PACKAGE_BUGREPORT = 
+PACKAGE_NAME = lxc
+PACKAGE_STRING = lxc 0.8.0-rc1
+PACKAGE_TARNAME = lxc
+PACKAGE_URL = 
+PACKAGE_VERSION = 0.8.0-rc1
+PATH_SEPARATOR = :
+PREFIX = /usr
+SETCAP = no
+SET_MAKE = 
+SHELL = /bin/bash
+STRIP = 
+SYSCONFDIR = /etc
+VERSION = 0.8.0-rc1
+abs_builddir = /home/sven/lxc-0.8.0~rc1/config
+abs_srcdir = /home/sven/lxc-0.8.0~rc1/config
+abs_top_builddir = /home/sven/lxc-0.8.0~rc1
+abs_top_srcdir = /home/sven/lxc-0.8.0~rc1
+ac_ct_CC = gcc
+am__include = include
+am__leading_dot = .
+am__quote = 
+am__tar = ${AMTAR} chof - "$$tardir"
+am__untar = ${AMTAR} xf -
+bindir = ${exec_prefix}/bin
+build = x86_64-pc-linux-gnu
+build_alias = x86_64-linux-gnu
+build_cpu = x86_64
+build_os = linux-gnu
+build_vendor = pc
+builddir = .
+datadir = ${datarootdir}
+datarootdir = ${prefix}/share
+docdir = ${datarootdir}/doc/${PACKAGE_TARNAME}
+dvidir = ${docdir}
+exec_prefix = ${prefix}
+have_docbook = yes
+host = x86_64-pc-linux-gnu
+host_alias = 
+host_cpu = x86_64
+host_os = linux-gnu
+host_vendor = pc
+htmldir = ${docdir}
+includedir = ${prefix}/include
+infodir = ${prefix}/share/info
+install_sh = ${SHELL} /home/sven/lxc-0.8.0~rc1/config/install-sh
+libdir = ${prefix}/lib/lxc
+libexecdir = ${prefix}/lib
+localedir = ${datarootdir}/locale
+localstatedir = /var
+mandir = ${prefix}/share/man
+mkdir_p = /bin/mkdir -p
+oldincludedir = /usr/include
+pdfdir = ${docdir}
+prefix = /usr
+program_transform_name = s,x,x,
+psdir = ${docdir}
+sbindir = ${exec_prefix}/sbin
+sharedstatedir = ${prefix}/com
+srcdir = .
+sysconfdir = /etc
+target_alias = 
+top_build_prefix = ../
+top_builddir = ..
+top_srcdir = ..
+all: all-am
+
+.SUFFIXES:
+$(srcdir)/Makefile.in:  $(srcdir)/Makefile.am  $(am__configure_deps)
+	@for dep in $?; do \
+	  case '$(am__configure_deps)' in \
+	    *$$dep*) \
+	      ( cd $(top_builddir) && $(MAKE) $(AM_MAKEFLAGS) am--refresh ) \
+	        && { if test -f $@; then exit 0; else break; fi; }; \
+	      exit 1;; \
+	  esac; \
+	done; \
+	echo ' cd $(top_srcdir) && $(AUTOMAKE) --gnu config/Makefile'; \
+	$(am__cd) $(top_srcdir) && \
+	  $(AUTOMAKE) --gnu config/Makefile
+.PRECIOUS: Makefile
+Makefile: $(srcdir)/Makefile.in $(top_builddir)/config.status
+	@case '$?' in \
+	  *config.status*) \
+	    cd $(top_builddir) && $(MAKE) $(AM_MAKEFLAGS) am--refresh;; \
+	  *) \
+	    echo ' cd $(top_builddir) && $(SHELL) ./config.status $(subdir)/$@ $(am__depfiles_maybe)'; \
+	    cd $(top_builddir) && $(SHELL) ./config.status $(subdir)/$@ $(am__depfiles_maybe);; \
+	esac;
+
+$(top_builddir)/config.status: $(top_srcdir)/configure $(CONFIG_STATUS_DEPENDENCIES)
+	cd $(top_builddir) && $(MAKE) $(AM_MAKEFLAGS) am--refresh
+
+$(top_srcdir)/configure:  $(am__configure_deps)
+	cd $(top_builddir) && $(MAKE) $(AM_MAKEFLAGS) am--refresh
+$(ACLOCAL_M4):  $(am__aclocal_m4_deps)
+	cd $(top_builddir) && $(MAKE) $(AM_MAKEFLAGS) am--refresh
+$(am__aclocal_m4_deps):
+tags: TAGS
+TAGS:
+
+ctags: CTAGS
+CTAGS:
+
+
+distdir: $(DISTFILES)
+	@srcdirstrip=`echo "$(srcdir)" | sed 's/[].[^$$\\*]/\\\\&/g'`; \
+	topsrcdirstrip=`echo "$(top_srcdir)" | sed 's/[].[^$$\\*]/\\\\&/g'`; \
+	list='$(DISTFILES)'; \
+	  dist_files=`for file in $$list; do echo $$file; done | \
+	  sed -e "s|^$$srcdirstrip/||;t" \
+	      -e "s|^$$topsrcdirstrip/|$(top_builddir)/|;t"`; \
+	case $$dist_files in \
+	  */*) $(MKDIR_P) `echo "$$dist_files" | \
+			   sed '/\//!d;s|^|$(distdir)/|;s,/[^/]*$$,,' | \
+			   sort -u` ;; \
+	esac; \
+	for file in $$dist_files; do \
+	  if test -f $$file || test -d $$file; then d=.; else d=$(srcdir); fi; \
+	  if test -d $$d/$$file; then \
+	    dir=`echo "/$$file" | sed -e 's,/[^/]*$$,,'`; \
+	    if test -d "$(distdir)/$$file"; then \
+	      find "$(distdir)/$$file" -type d ! -perm -700 -exec chmod u+rwx {} \;; \
+	    fi; \
+	    if test -d $(srcdir)/$$file && test $$d != $(srcdir); then \
+	      cp -fpR $(srcdir)/$$file "$(distdir)$$dir" || exit 1; \
+	      find "$(distdir)/$$file" -type d ! -perm -700 -exec chmod u+rwx {} \;; \
+	    fi; \
+	    cp -fpR $$d/$$file "$(distdir)$$dir" || exit 1; \
+	  else \
+	    test -f "$(distdir)/$$file" \
+	    || cp -p $$d/$$file "$(distdir)/$$file" \
+	    || exit 1; \
+	  fi; \
+	done
+check-am: all-am
+check: check-am
+all-am: Makefile
+installdirs:
+install: install-am
+install-exec: install-exec-am
+install-data: install-data-am
+uninstall: uninstall-am
+
+install-am: all-am
+	@$(MAKE) $(AM_MAKEFLAGS) install-exec-am install-data-am
+
+installcheck: installcheck-am
+install-strip:
+	$(MAKE) $(AM_MAKEFLAGS) INSTALL_PROGRAM="$(INSTALL_STRIP_PROGRAM)" \
+	  install_sh_PROGRAM="$(INSTALL_STRIP_PROGRAM)" INSTALL_STRIP_FLAG=-s \
+	  `test -z '$(STRIP)' || \
+	    echo "INSTALL_PROGRAM_ENV=STRIPPROG='$(STRIP)'"` install
+mostlyclean-generic:
+
+clean-generic:
+
+distclean-generic:
+	-test -z "$(CONFIG_CLEAN_FILES)" || rm -f $(CONFIG_CLEAN_FILES)
+	-test . = "$(srcdir)" || test -z "$(CONFIG_CLEAN_VPATH_FILES)" || rm -f $(CONFIG_CLEAN_VPATH_FILES)
+
+maintainer-clean-generic:
+	@echo "This command is intended for maintainers to use"
+	@echo "it deletes files that may require special tools to rebuild."
+clean: clean-am
+
+clean-am: clean-generic mostlyclean-am
+
+distclean-am: clean-am distclean-generic
+
+dvi: dvi-am
+
+dvi-am:
+
+html: html-am
+
+html-am:
+
+info: info-am
+
+info-am:
+
+install-data-am:
+
+install-dvi: install-dvi-am
+
+install-dvi-am:
+
+install-exec-am:
+
+install-html: install-html-am
+
+install-html-am:
+
+install-info: install-info-am
+
+install-info-am:
+
+install-man:
+
+install-pdf: install-pdf-am
+
+install-pdf-am:
+
+install-ps: install-ps-am
+
+install-ps-am:
+
+installcheck-am:
+
+maintainer-clean: maintainer-clean-am
+	-rm -f Makefile
+maintainer-clean-am: distclean-am maintainer-clean-generic
+
+mostlyclean: mostlyclean-am
+
+mostlyclean-am: mostlyclean-generic
+
+pdf: pdf-am
+
+pdf-am:
+
+ps: ps-am
+
+ps-am:
+
+uninstall-am:
+
+.MAKE: install-am install-strip
+
+.PHONY: all all-am check check-am clean clean-generic distclean \
+	distclean-generic distdir dvi dvi-am html html-am info info-am \
+	install install-am install-data install-data-am install-dvi \
+	install-dvi-am install-exec install-exec-am install-html \
+	install-html-am install-info install-info-am install-man \
+	install-pdf install-pdf-am install-ps install-ps-am \
+	install-strip installcheck installcheck-am installdirs \
+	maintainer-clean maintainer-clean-generic mostlyclean \
+	mostlyclean-generic pdf pdf-am ps ps-am uninstall uninstall-am
+
+maintainerclean:
+	@$(RM) -f compile config.guess config.sub depcomp install-sh ltmain.sh missing Makefile.in Makefile
+
+# Tell versions [3.59,3.63) of GNU make to not export all variables.
+# Otherwise a system limit (for SysV at least) may be exceeded.
+.NOEXPORT:
--- lxc-0.8.0~rc1.orig/doc/lxc-execute.1
+++ lxc-0.8.0~rc1/doc/lxc-execute.1
@@ -3,7 +3,7 @@
 .\" <http://shell.ipoline.com/~elmert/comp/docbook2X/> 
 .\" Please send any bug reports, improvements, comments, patches, 
 .\" etc. to Steve Cheng <steve@ggi-project.org>.
-.TH "LXC-EXECUTE" "1" "02 March 2012" "" ""
+.TH "LXC-EXECUTE" "1" "06 July 2012" "" ""
 
 .SH NAME
 lxc-execute \- run an application inside a container.
--- lxc-0.8.0~rc1.orig/doc/lxc-destroy.1
+++ lxc-0.8.0~rc1/doc/lxc-destroy.1
@@ -3,7 +3,7 @@
 .\" <http://shell.ipoline.com/~elmert/comp/docbook2X/> 
 .\" Please send any bug reports, improvements, comments, patches, 
 .\" etc. to Steve Cheng <steve@ggi-project.org>.
-.TH "LXC-DESTROY" "1" "02 March 2012" "" ""
+.TH "LXC-DESTROY" "1" "06 July 2012" "" ""
 
 .SH NAME
 lxc-destroy \- destroy a container.
--- lxc-0.8.0~rc1.orig/doc/lxc-ls.1
+++ lxc-0.8.0~rc1/doc/lxc-ls.1
@@ -3,7 +3,7 @@
 .\" <http://shell.ipoline.com/~elmert/comp/docbook2X/> 
 .\" Please send any bug reports, improvements, comments, patches, 
 .\" etc. to Steve Cheng <steve@ggi-project.org>.
-.TH "LXC-LS" "1" "02 March 2012" "" ""
+.TH "LXC-LS" "1" "06 July 2012" "" ""
 
 .SH NAME
 lxc-ls \- list the containers existing on the system
--- lxc-0.8.0~rc1.orig/doc/lxc-wait.1
+++ lxc-0.8.0~rc1/doc/lxc-wait.1
@@ -3,7 +3,7 @@
 .\" <http://shell.ipoline.com/~elmert/comp/docbook2X/> 
 .\" Please send any bug reports, improvements, comments, patches, 
 .\" etc. to Steve Cheng <steve@ggi-project.org>.
-.TH "LXC-WAIT" "1" "02 March 2012" "" ""
+.TH "LXC-WAIT" "1" "06 July 2012" "" ""
 
 .SH NAME
 lxc-wait \- wait for a specific container state
--- lxc-0.8.0~rc1.orig/doc/lxc-start.1
+++ lxc-0.8.0~rc1/doc/lxc-start.1
@@ -3,7 +3,7 @@
 .\" <http://shell.ipoline.com/~elmert/comp/docbook2X/> 
 .\" Please send any bug reports, improvements, comments, patches, 
 .\" etc. to Steve Cheng <steve@ggi-project.org>.
-.TH "LXC-START" "1" "02 March 2012" "" ""
+.TH "LXC-START" "1" "06 July 2012" "" ""
 
 .SH NAME
 lxc-start \- run an application inside a container.
--- lxc-0.8.0~rc1.orig/doc/lxc-checkpoint.1
+++ lxc-0.8.0~rc1/doc/lxc-checkpoint.1
@@ -3,7 +3,7 @@
 .\" <http://shell.ipoline.com/~elmert/comp/docbook2X/> 
 .\" Please send any bug reports, improvements, comments, patches, 
 .\" etc. to Steve Cheng <steve@ggi-project.org>.
-.TH "LXC-CHECKPOINT" "1" "02 March 2012" "IBM" ""
+.TH "LXC-CHECKPOINT" "1" "06 July 2012" "IBM" ""
 
 .SH NAME
 lxc-checkpoint \- checkpoint a running container (not implemented yet)
--- /dev/null
+++ lxc-0.8.0~rc1/doc/manpage.refs
@@ -0,0 +1,4 @@
+{
+  '' => '',
+  '' => ''
+}
--- lxc-0.8.0~rc1.orig/doc/lxc-cgroup.1
+++ lxc-0.8.0~rc1/doc/lxc-cgroup.1
@@ -3,7 +3,7 @@
 .\" <http://shell.ipoline.com/~elmert/comp/docbook2X/> 
 .\" Please send any bug reports, improvements, comments, patches, 
 .\" etc. to Steve Cheng <steve@ggi-project.org>.
-.TH "LXC-CGROUP" "1" "02 March 2012" "" ""
+.TH "LXC-CGROUP" "1" "06 July 2012" "" ""
 
 .SH NAME
 lxc-cgroup \- manage the control group associated with a container
--- lxc-0.8.0~rc1.orig/doc/lxc-unfreeze.1
+++ lxc-0.8.0~rc1/doc/lxc-unfreeze.1
@@ -3,7 +3,7 @@
 .\" <http://shell.ipoline.com/~elmert/comp/docbook2X/> 
 .\" Please send any bug reports, improvements, comments, patches, 
 .\" etc. to Steve Cheng <steve@ggi-project.org>.
-.TH "LXC-UNFREEZE" "1" "02 March 2012" "" ""
+.TH "LXC-UNFREEZE" "1" "06 July 2012" "" ""
 
 .SH NAME
 lxc-unfreeze \- thaw all the container's processes
--- lxc-0.8.0~rc1.orig/doc/lxc-monitor.1
+++ lxc-0.8.0~rc1/doc/lxc-monitor.1
@@ -3,7 +3,7 @@
 .\" <http://shell.ipoline.com/~elmert/comp/docbook2X/> 
 .\" Please send any bug reports, improvements, comments, patches, 
 .\" etc. to Steve Cheng <steve@ggi-project.org>.
-.TH "LXC-MONITOR" "1" "02 March 2012" "" ""
+.TH "LXC-MONITOR" "1" "06 July 2012" "" ""
 
 .SH NAME
 lxc-monitor \- monitor the container state
--- lxc-0.8.0~rc1.orig/doc/lxc-stop.1
+++ lxc-0.8.0~rc1/doc/lxc-stop.1
@@ -3,7 +3,7 @@
 .\" <http://shell.ipoline.com/~elmert/comp/docbook2X/> 
 .\" Please send any bug reports, improvements, comments, patches, 
 .\" etc. to Steve Cheng <steve@ggi-project.org>.
-.TH "LXC-STOP" "1" "02 March 2012" "" ""
+.TH "LXC-STOP" "1" "06 July 2012" "" ""
 
 .SH NAME
 lxc-stop \- stop the application running inside a container
--- lxc-0.8.0~rc1.orig/doc/lxc-restart.1
+++ lxc-0.8.0~rc1/doc/lxc-restart.1
@@ -3,7 +3,7 @@
 .\" <http://shell.ipoline.com/~elmert/comp/docbook2X/> 
 .\" Please send any bug reports, improvements, comments, patches, 
 .\" etc. to Steve Cheng <steve@ggi-project.org>.
-.TH "LXC-RESTART" "1" "02 March 2012" "IBM" ""
+.TH "LXC-RESTART" "1" "06 July 2012" "IBM" ""
 
 .SH NAME
 lxc-restart \- restart a container from a file (not implemented yet)
--- lxc-0.8.0~rc1.orig/doc/lxc.7
+++ lxc-0.8.0~rc1/doc/lxc.7
@@ -3,7 +3,7 @@
 .\" <http://shell.ipoline.com/~elmert/comp/docbook2X/> 
 .\" Please send any bug reports, improvements, comments, patches, 
 .\" etc. to Steve Cheng <steve@ggi-project.org>.
-.TH "LXC" "7" "02 March 2012" "Version 0.8.0-rc1" ""
+.TH "LXC" "7" "06 July 2012" "Version 0.8.0-rc1" ""
 
 .SH NAME
 lxc \- linux containers
--- lxc-0.8.0~rc1.orig/doc/lxc-kill.1
+++ lxc-0.8.0~rc1/doc/lxc-kill.1
@@ -3,7 +3,7 @@
 .\" <http://shell.ipoline.com/~elmert/comp/docbook2X/> 
 .\" Please send any bug reports, improvements, comments, patches, 
 .\" etc. to Steve Cheng <steve@ggi-project.org>.
-.TH "LXC-KILL" "1" "02 March 2012" "IBM" ""
+.TH "LXC-KILL" "1" "06 July 2012" "IBM" ""
 
 .SH NAME
 lxc-kill \- Send a signal to the process 1 of the container.
--- lxc-0.8.0~rc1.orig/doc/lxc.conf.5
+++ lxc-0.8.0~rc1/doc/lxc.conf.5
@@ -3,7 +3,7 @@
 .\" <http://shell.ipoline.com/~elmert/comp/docbook2X/> 
 .\" Please send any bug reports, improvements, comments, patches, 
 .\" etc. to Steve Cheng <steve@ggi-project.org>.
-.TH "LXC.CONF" "5" "02 March 2012" "" ""
+.TH "LXC.CONF" "5" "06 July 2012" "" ""
 
 .SH NAME
 lxc.conf \- linux container configuration file
@@ -254,7 +254,7 @@ mount informations. If the rootfs is an
 device block and the fstab is used to mount a point
 somewhere in this rootfs, the path of the rootfs mount
 point should be prefixed with the
-\fI${libdir}/lxc/rootfs\fR default path or
+\fI${prefix}/lib/lxc\fR default path or
 the value of \fBlxc.rootfs.mount\fR if
 specified.
 .TP
@@ -334,7 +334,7 @@ eth0.
 	lxc.network.link = br0
 	lxc.network.name = eth0
 	lxc.network.hwaddr = 4a:49:43:49:79:bf
-	lxc.network.ipv4 = 1.2.3.5/24 1.2.3.255
+	lxc.network.ipv4 = 10.2.3.5/24 10.2.3.255
 	lxc.network.ipv6 = 2003:db8:1:0:214:1234:fe0b:3597
       
 .fi
@@ -365,21 +365,21 @@ mounting some locations and a changing r
 	lxc.network.flags = up
 	lxc.network.link = br0
 	lxc.network.hwaddr = 4a:49:43:49:79:bf
-	lxc.network.ipv4 = 1.2.3.5/24 1.2.3.255
+	lxc.network.ipv4 = 10.2.3.5/24 10.2.3.255
 	lxc.network.ipv6 = 2003:db8:1:0:214:1234:fe0b:3597
 	lxc.network.ipv6 = 2003:db8:1:0:214:5432:feab:3588
 	lxc.network.type = macvlan
 	lxc.network.flags = up
 	lxc.network.link = eth0
 	lxc.network.hwaddr = 4a:49:43:49:79:bd
-	lxc.network.ipv4 = 1.2.3.4/24
+	lxc.network.ipv4 = 10.2.3.4/24
 	lxc.network.ipv4 = 192.168.10.125/24
 	lxc.network.ipv6 = 2003:db8:1:0:214:1234:fe0b:3596
 	lxc.network.type = phys
 	lxc.network.flags = up
 	lxc.network.link = dummy0
 	lxc.network.hwaddr = 4a:49:43:49:79:ff
-	lxc.network.ipv4 = 1.2.3.6/24
+	lxc.network.ipv4 = 10.2.3.6/24
 	lxc.network.ipv6 = 2003:db8:1:0:214:1234:fe0b:3297
 	lxc.cgroup.cpuset.cpus = 0,1
 	lxc.cgroup.cpu.shares = 1234
--- lxc-0.8.0~rc1.orig/doc/lxc-console.1
+++ lxc-0.8.0~rc1/doc/lxc-console.1
@@ -3,7 +3,7 @@
 .\" <http://shell.ipoline.com/~elmert/comp/docbook2X/> 
 .\" Please send any bug reports, improvements, comments, patches, 
 .\" etc. to Steve Cheng <steve@ggi-project.org>.
-.TH "LXC-CONSOLE" "1" "02 March 2012" "" ""
+.TH "LXC-CONSOLE" "1" "06 July 2012" "" ""
 
 .SH NAME
 lxc-console \- Launch a console for the specified container
--- lxc-0.8.0~rc1.orig/doc/lxc-attach.1
+++ lxc-0.8.0~rc1/doc/lxc-attach.1
@@ -3,7 +3,7 @@
 .\" <http://shell.ipoline.com/~elmert/comp/docbook2X/> 
 .\" Please send any bug reports, improvements, comments, patches, 
 .\" etc. to Steve Cheng <steve@ggi-project.org>.
-.TH "LXC-ATTACH" "1" "02 March 2012" "" ""
+.TH "LXC-ATTACH" "1" "06 July 2012" "" ""
 
 .SH NAME
 lxc-attach \- start a process inside a running container.
--- lxc-0.8.0~rc1.orig/doc/lxc-freeze.1
+++ lxc-0.8.0~rc1/doc/lxc-freeze.1
@@ -3,7 +3,7 @@
 .\" <http://shell.ipoline.com/~elmert/comp/docbook2X/> 
 .\" Please send any bug reports, improvements, comments, patches, 
 .\" etc. to Steve Cheng <steve@ggi-project.org>.
-.TH "LXC-FREEZE" "1" "02 March 2012" "" ""
+.TH "LXC-FREEZE" "1" "06 July 2012" "" ""
 
 .SH NAME
 lxc-freeze \- freeze all the container's processes
--- lxc-0.8.0~rc1.orig/doc/lxc-create.1
+++ lxc-0.8.0~rc1/doc/lxc-create.1
@@ -3,7 +3,7 @@
 .\" <http://shell.ipoline.com/~elmert/comp/docbook2X/> 
 .\" Please send any bug reports, improvements, comments, patches, 
 .\" etc. to Steve Cheng <steve@ggi-project.org>.
-.TH "LXC-CREATE" "1" "02 March 2012" "" ""
+.TH "LXC-CREATE" "1" "06 July 2012" "" ""
 
 .SH NAME
 lxc-create \- creates a container
@@ -42,7 +42,7 @@ and isolation functionalities for the co
 \&'template' is the short name of an existing 'lxc-template'
 script that is called by lxc-create,
 eg. busybox, debian, fedora, ubuntu or sshd.
-Refer to the examples in \fI${libdir}/lxc/templates\fR
+Refer to the examples in \fI/usr/share/lxc/templates\fR
 for details of the expected script structure.
 .SH "DIAGNOSTIC"
 .TP
--- lxc-0.8.0~rc1.orig/doc/lxc-ps.1
+++ lxc-0.8.0~rc1/doc/lxc-ps.1
@@ -3,7 +3,7 @@
 .\" <http://shell.ipoline.com/~elmert/comp/docbook2X/> 
 .\" Please send any bug reports, improvements, comments, patches, 
 .\" etc. to Steve Cheng <steve@ggi-project.org>.
-.TH "LXC-PS" "1" "02 March 2012" "" ""
+.TH "LXC-PS" "1" "06 July 2012" "" ""
 
 .SH NAME
 lxc-ps \- list the processes belonging to a specific container.
